.github/renovate.json5

```diff
@@ -0,0 +1,76 @@
+{
+  extends: [
+    "config:base",
+    "schedule:earlyMondays"
+  ],
+  automerge: false,
+  platformAutomerge: false,
+  labels: ["dependencies"],
+  postUpdateOptions: ["gomodTidy", "yarnDedupeHighest"],
+  prHourlyLimit: 0,
+  prConcurrentLimit: 5,
+  packageRules: [
+    // e2e test depends on testing/Dockefile testing-image which has conftest specific version.
+    // to upgrade conftest versions, we need following PRs.
+    // 1. update testing/Dockerfile conftest version
+    // 2. update testing-env tag
+    // 3. update e2e conftest version
+    // this settings allow to create only testing/Dockefile contest version update PR which uses branch prefix.
+    {
+      matchPaths: ["testing/**"],
+      matchPackagePatterns: ["conftest"],
+      additionalBranchPrefix: "{{baseDir}}-",
+      groupName: "conftest-testing",
+      prBodyNotes: [
+        ":warning: You need to upgrade testing-env conftest firstly, then upgrade other conftest versions for e2e :warning:",
+      ],
+    },
+    {
+      ignorePaths: ["testing/**"],
+      matchPackagePatterns: ["github-actions"],
+      groupName: "github-",
+    },
+    {
+      // we need to upgrade testing-env on ci quickly
+      matchPaths: [".github/**"],
+      matchPackageNames: ["ghcr.io/runatlantis/testing-env"],
+      groupName: "testing-env-ci-test",
+      schedules: ["every 1 hour after 00:00 and before 23:59 every day"],
```

Comment:
oops, this schedules doesn't works well, wait a sec 🙇

Reply:
```suggestion
      schedules ["every 1 hour after 00:00 and before 23:59 every day"],
```

> npx --package renovate -c 'renovate-config-validator'

<img width="270" alt="image" src="https://user-images.githubusercontent.com/915731/208323353-b14f0e36-8ecd-467b-b727-996c18e40f08.png">